pkg/controllers/provisioning/provisioner.go

```diff
@@ -157,30 +157,15 @@ func (p *Provisioner) launch(ctx context.Context, constraints *v1alpha5.Constrai
 		pods <- ps
 	}
 	return p.cloudProvider.Create(ctx, constraints, packing.InstanceTypeOptions, packing.NodeQuantity, func(node *v1.Node) error {
-		node.Labels = functional.UnionStringMaps(node.Labels, constraints.GenerateLabels())
-		node.Spec.Taints = append(node.Spec.Taints, constraints.Taints...)
-		return p.bind(ctx, node, <-pods)
+		n := constraints.ToNode()
```

Comment:
Did we drop the resources that the cloud provider added to the node here?  (mergo wouldn't have, sorry :( )

Reply:
Ah, they're in .Status.